modules/acl-controller/README.md

```diff
@@ -0,0 +1,6 @@
+# ACL Controller
```

Comment:
Do we forsee any other uses for this? Wondering if we should just call it controller.

Reply:
We can potentially re-use the code in consul-ecs but I feel like the tf module is specific to ACLs in terms of the variables it takes and outputs it produces. It's also hard to predict what we'll do/need in the future, but I think if we need to rename it and/or make it more generic, it should definitely possible.

For now, I'm inclined to keep it specific. WDYT?